Templates/VSIX/Project Templates/cpp-winui-template/WinUI3TemplateCpp/SampleConfiguration.cpp

```diff
@@ -6,16 +6,20 @@
 #include "SampleConfiguration.h"
 #include "MainPage.xaml.h" 
 
-using namespace winrt;
-using namespace winrt::$safeprojectname$;
-using namespace Microsoft::UI::Xaml;
-using namespace Windows::Foundation::Collections;
+namespace winrt
+{
+    using namespace Microsoft::UI::Xaml;
+    using namespace Windows::Foundation::Collections;
+}
 
-IVector<Scenario> implementation::MainPage::scenariosInner = winrt::single_threaded_observable_vector<Scenario>(
-	{
-		Scenario{ L"Scenario 1", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario1_ShortName>())},
-		Scenario{ L"Scenario 2", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario2_ShortName>())}
-	});
+namespace winrt::$safeprojectname$
+{
+    IVector<Scenario> implementation::MainPage::scenariosInner = winrt::single_threaded_observable_vector<Scenario>(
+        {
+            Scenario{ L"Scenario 1", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario1_ShortName>())},
+            Scenario{ L"Scenario 2", winrt::hstring(winrt::name_of<$safeprojectname$::Scenario2_ShortName>())}
```

Comment:
Huh, I'm surprised you need this constructor here - what happens if you just say:

```c++
Scenario{ L"Scenario 1", winrt::name_of<$safeprojectname$::Scenario1_ShortName>()}
```

Reply:
When I tried that, I got the error: 
E0312	no suitable user-defined conversion from "std::basic_string_view<wchar_t, std::char_traits<wchar_t>>" to "winrt::hstring" exists	

Probably because `Project.idl` defines Scenario struct as follows:
```idl
struct Scenario
{
     String Title;
     String ClassName;
    };
```